docset/winserver2022-ps/defender/Get-PerformanceReport.md

```diff
@@ -0,0 +1,331 @@
+---
+description: Use this topic to analyze and report on Microsoft Defender Antivirus (MDAV) performance recordings.
+external help file: MSFT_MpThreat.cdxml-help.xml
+Module Name: Defender
+ms.date: 03/20/2022
+online version: 
+schema: 2.0.0
+title: Get-MpPerformanceReport
+
+---
+
+# Get-MpPerformanceReport

```

Comment:
The uploaded file name says "Get-PerformanceReport"

Reply:
Changed

Reply:
changed